### DISCUSSION TITLE
CorsixTH 0.65.1 Hotfix

### DISCUSSION THREAD

**Discussion by TheCycoONE** [CorsixTH/CorsixTH#2355] [Team Posts]
There is a hotfix tag to fix a crash bug when running a save game from a 0.64 or earlier save in 0.65 and getting the new no deaths trophy.

The tag is v0.65.1, I will be publishing the release today. @colinjmatt if you have a chance, I think it would be good to have the mac version out. A new compile isn't needed, the only changes are in lua.

**Comment by colinjmatt** [comment]
I’ll have the Mac package out in about 4 hours. It would be sooner but I’ve not been home all day

**Comment by TheCycoONE** [comment]
No problem. Thanks!

**Comment by colinjmatt** [comment]
Uploaded :)

**Comment by rmnvgr** [comment]
Flatpak update is ready to be merged and published.

**Comment by TheCycoONE** [comment]
Published the release. I'll add the signature tonight
